src/style.css

```diff
@@ -1,3 +1,80 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  --active-link-color: #00acdc;
+  --static-font-color: #000;
```

Comment:
good practice to declare variables inside the [:root](https://www.w3schools.com/css/css3_variables.asp) selector
```css
:root {
 --active-link-color: #00acdc;
 --static-font-color: #000;
}
```